### ISSUE TITLE
PP-OCRv3 Mobile Detection hmean saturates at ~0.51 despite 500 epochs training on custom dataset

### ISSUE THREAD

**Issue by Hanumeshkulklkarni** [PaddlePaddle/PaddleOCR#18059]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

Hello PaddleOCR Team,

I am training a PP-OCRv3 mobile detection model (DB detector with MobileNetV3 backbone) on a custom merged dataset, but the hmean metric saturates very early around 0.51–0.52 and does not improve even after long training.

Environment:

PaddleOCR version: latest v3
Model: PP-OCRv3_mobile_det
GPU: NVIDIA GPU
OS: Windows
Training from pretrained student weights

Dataset:
I merged multiple datasets together:

SynthText_F33
SynthText_F154
SynthText_F15
SynthText_F20
TextOCR
TotalText
Arbitrary
ICDAR15
LSVT
MLT19

Total samples: ~66k

Training observations:

Training is stable
No errors or warnings
Loss decreases normally
Best epoch was around 136
Hmean remains around 0.51–0.52 even after 500 epochs
Precision improves slightly but recall remains low (~0.45)

Best metrics:

Hmean: 0.52
Precision: 0.58
Recall: 0.45

I also tried:

continuing training from best_accuracy model
increasing epochs from 100 → 200 → 500
changing synthetic/real dataset ratios

Current dataset ratio used:

initially 75% real / 25% synthetic
planning to test 60:40 and 50:50

### 🏃‍♂️ Environment (运行环境)

```bash
OS:
Microsoft Windows 11 Home Single Language
Version: 10.0.26200 Build 26200

Python:
3.10.9

PaddlePaddle:
2.5.2.post117

PaddleOCR:
Installed from GitHub source

CUDA:
12.5

GPU:
NVIDIA GeForce GTX 1650 (4 GB VRAM)

CPU:
Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz

RAM:
8 GB

Install method:
git
```

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

Questions:

Is this saturation expected for MobileNetV3 scale=0.5?
Could mixed datasets with different annotation styles cause this plateau?
Is my augmentation pipeline too aggressive?
Should I change backbone scale or learning rate strategy?
Is continuing training from best_accuracy model recommended for DB detector training?

My current config is attached below.

<img width="1581" height="582" alt="Image" src="https://github.com/user-attachments/assets/98fcabb3-4d81-4127-8c58-1d011ec0bc72" />
<img width="1644" height="566" alt="Image" src="https://github.com/user-attachments/assets/08a8ff11-29a1-4d71-895a-c8d214783f63" />
<img width="1608" height="597" alt="Image" src="https://github.com/user-attachments/assets/7154e26c-7ad8-420d-96ae-f47abcef84ad" />
<img width="1636" height="670" alt="Image" src="https://github.com/user-attachments/assets/32accedb-9bde-4aab-8a59-980583027fe9" />

I have also attached training log screenshots.

Any suggestions would be very helpful.
Thank you

**Comment by SISTMrL** [comment]
主要原因：

多数据集标注粒度不一致：SynthText / TextOCR / TotalText / ICDAR15 / LSVT / MLT19 的标注粒度差异很大（词级 vs 行级 vs 段级）。DB 检测对标注一致性非常敏感，混合不同粒度会让模型学不到统一的「文本实例」定义，直接压低 recall。这是最可能的主因。
backbone 太小：MobileNetV3 scale=0.5 容量有限，在如此多样的数据分布上 ~0.5 的 hmean 属于能力上限附近。
显存约束导致 batch 过小：GTX 1650 4GB，batch size 只能很小，DB 训练对 batch size 敏感，太小会影响 BN 和收敛质量。
建议：

优先统一标注粒度（最重要）：所有数据集统一到行级或统一到词级，不要混用。
先只用 1~2 个高质量真实数据集（如 ICDAR15 + LSVT）跑通，确认能到合理 hmean，再逐步加数据。
换 PP-OCRv5_mobile_det 或至少 MobileNetV3 scale=1.0 提升容量。
从 best_accuracy 断点继续训练用于 finetune 是可以的，但如果数据/配置有根本问题，继续训练不会突破瓶颈。
检查后处理 unclip_ratio、box_thresh——recall 低时可适当降低 box_thresh、增大 unclip_ratio 观察
